lib/password_strategies/woof_bark.rb

```diff
@@ -0,0 +1,28 @@
+module PasswordStrategies
```

Comment:
this is our home rolled password strategy, once this is out we can force a forgot password and swap it to the default bcrypt version

Reply:
Will do!